torch/fx/experimental/symbolic_shapes.py

```diff
@@ -216,6 +216,8 @@ def free_symbols(val: Union[SymInt, torch.Tensor]) -> Set[sympy.Symbol]:
         for s in val:
             r |= free_symbols(s)
         return r
+    elif isinstance(val, functools.partial):
+        return set()
```

Comment:
this is suspicious. Because functools.partial can contain arbitrary objects, which might themselves have free symbols.